WasatchNET/EEPROM.cs

```diff
@@ -750,9 +750,10 @@ public string userText
             set
             {
                 EventHandler handler = EEPROMChanged;
-                userData = new byte[value.Length];
+                userData = new byte[value.Length + 1];

```

Comment:
Should we cap this at 64 bytes?

Reply:
with the "user data" page 6/7 format it'd need to be 192 (I think) and still wouldn't play nice with the other formats if the user went over 64 so I think I'm happy to give the user the responsibility not to use too many bytes